DqtScripts/DqtScripts/Program.cs

```diff
@@ -82,10 +82,66 @@ static RootCommand CreateRootCommand()
     AddCleanseTraineeIdCommand(rootCommand);
     AddCombineHesaAndDmsTeacherStatusesCommand(rootCommand);
     AddCombineRenameDMSTraineeTeacherStatusCommand(rootCommand);
+    AddMasterChemistryMasterPhysicsQualifications(rootCommand);
 
     return rootCommand;
 }
 
+static void AddMasterChemistryMasterPhysicsQualifications(RootCommand rootCommand)
+{
+    var command = new Command("add-physics-chemistry-qualifcations", description: "Adds Master Physics & Master Chemisty & Bachelor of Arts in Combined Studies HE Qualifications")
+    {
+        Handler = CommandHandler.Create<IHost, bool?>(async (host, commit) =>
+        {
+            var serviceClient = host.Services.GetRequiredService<ServiceClient>();
+
+            //Fetch existing
+            var qualificationsQuery = new QueryExpression("dfeta_hequalification");
+            qualificationsQuery.Criteria.AddCondition("dfeta_value", ConditionOperator.In, new string[] { "214", "215","009" });
+            qualificationsQuery.ColumnSet = new ColumnSet("dfeta_value", "dfeta_name");
+            qualificationsQuery.PageInfo = new PagingInfo()
+            {
+                Count = 3,
+                PageNumber = 1
+            };
+            var heQualifications = await serviceClient.RetrieveMultipleAsync(qualificationsQuery);
+            var existingMoChemistry = heQualifications.Entities.SingleOrDefault(x => x["dfeta_value"].ToString() == "214");
+            var existingMoPhysics = heQualifications.Entities.SingleOrDefault(x => x["dfeta_value"].ToString() == "215");
+            var existingBoArts = heQualifications.Entities.SingleOrDefault(x => x["dfeta_value"].ToString() == "009");
+
+
```

Comment:
For each of these we should set a fixed `id` GUID so the IDs are the same over every environment.